### ISSUE TITLE
Update requirements file

### ISSUE THREAD

**Issue by ItamarWilf** [OperationalBina/PipeRT#136]
Are all the packages in the requirements.txt needed? When did we last prune the file?

**Comment by MayoG** [comment]
I don't think that all of the packages are needed, some of them are being used in the contrib folder in components that are outdated. After cleaning the reposirtory from all of the outdated code in the contrib, we will be able to tell for sure which package is needed and which is not.

We didn't prune the file yet.
